modular/gater/metadata_handler.go

```diff
@@ -2281,6 +2281,94 @@ func (g *GateModular) getUserOwnedGroupsHandler(w http.ResponseWriter, r *http.R
 	w.Write(respBytes)
 }
 
+// listObjectPoliciesHandler list policies by object info
+func (g *GateModular) listObjectPoliciesHandler(w http.ResponseWriter, r *http.Request) {
+	var (
+		err               error
+		respBytes         []byte
+		reqCtx            *RequestContext
+		limit             uint32
+		requestLimit      string
+		policies          []*types.Policy
+		requestActionType string
+		actionType        uint32
+		requestStartAfter string
+		startAfter        uint64
+		ok                bool
+		queryParams       url.Values
+	)
+
+	defer func() {
+		reqCtx.Cancel()
+		if err != nil {
+			reqCtx.SetError(gfsperrors.MakeGfSpError(err))
+			log.CtxErrorw(reqCtx.Context(), "failed to list policies by object info", reqCtx.String())
+			MakeErrorResponse(w, err)
```

Comment:
Please add in the metrics monitor in new handlers.

Reply:
fixed